collections/JAXA_wq_chla.yaml

```diff
@@ -4,6 +4,18 @@ Description: |
   It is the ratio (percentage) of weekly chlorophyll-a concentration divided by average concentration of other years for 4 sites:  North Adriatic, Tokyo, Kobe and Nagoya.  
   Anomaly[%]=((DN-1)/254*(300-(-100))-100).  
   The base data is made by averaging within -1, 0, +1 weeks in 2018-2020.
+  
+  File naming convention:
+  jx_chla_tif_XXX_yyyy_mm_dd.tif
+  XXX: Area name
+           nas  ->  NAdriatic
+           tok   ->  Tokyo 
+           kob   ->  Kobe 
+           nag   ->  Nagoya 
+  yyyy:Year
+  mm:Month
+  dd:Day
+
```

Comment:
to maintain the multiline fomat of the block of text;

I recommend to  place triple backticks (```) on the lines before and after the block, following  markdown synthax. 

it will then appear like this

  ```
  File naming convention: jx_tsm_tif_XXX_yyyy_mm_dd.tif
  XXX: Area name
           nas  ->  NAdriatic
           tok   ->  Tokyo 
           kob   ->  Kobe 
           nag   ->  Nagoya 
  yyyy:Year
  mm:Month
  dd:Day
  ```